packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -206,22 +193,21 @@ const SideNavFooter = styled("div", {
   }) => ({
     position: "sticky",
     bottom: 0,
-    paddingTop: odysseyDesignTokens.Spacing2,
     transitionProperty: "box-shadow",
     transitionDuration: odysseyDesignTokens.TransitionDurationMain,
     transitionTiming: odysseyDesignTokens.TransitionTimingMain,
+    backgroundColor: odysseyDesignTokens.HueNeutralWhite,
     // The box shadow should appear above the footer only if the scrollable region has overflow
     ...(isContentScrollable && {
-      boxShadow: odysseyDesignTokens.DepthHigh,
+      boxShadow: "0px -8px 8px -8px rgba(39, 39, 39, 0.08)",
```

Comment:
Is there a reason we couldn't use the shadow from Odyssey itself?

We can add `clip-path` if this was causing bleed-over.

Reply:
Do we need any support for IE11? Will SIW team use this?